client/src/components/Navbar.js

```diff
@@ -39,10 +42,10 @@ class Navbar extends React.Component {
       return (
         <>
           <Menu.Item link>
-          <Link to = '/login' class = ''>LOGIN</Link>
+            <Link to = '/login' className = ''>LOGIN</Link>
```

Comment:
Do we need these classNames?

Reply:
i can just remove them if you dont want them. i think joe put in class to start with